app/plugins/placeholdifier.js

```diff
@@ -0,0 +1,21 @@
+import { loadStyles } from '../utilities/styles.js'
+
+export const commands = [
+    'placeholdifier',
+]
+
+export const description = 'turn the page into a live wireframe'
+
+export default async function() {
+    await loadStyles(['https://unpkg.com/placeholdifier/placeholdifier.css'])
```

Comment:
```suggestion
    const stylesheet = document.createElement('link')
    stylesheet.setAttribute('rel', 'stylesheet')
    stylesheet.setAttribute('href', 'https://unpkg.com/placeholdifier/placeholdifier.css')
```

the clue here was that when the stylesheet was server referenced, the font paths resolve, but not if the inner text of the stylesheet is put in the page (like what `loadStyles()` does). This change create a link tag and puts it into the head, and all is well in the cosmos.

Reply:
nice find! that makes much more sense, its easier to understand why the locally served fonts wouldn't resolve if they are put in the page like how `loadStyles()` does it. Thanks for the help!